r2i/iprediction.h

```diff
@@ -27,19 +29,42 @@ class IPrediction {
  public:
   /**
    * \brief Gets the prediction for a particular index.
+   * \param output_index corresponding to the model output.
    * \param index Index for a value on the prediction matrix.
    * \param error [out] RuntimeError with a description of an error.
    * \return a double that indicates the prediction at the provided index .
    */
-  virtual double At (unsigned int index,  r2i::RuntimeError &error) = 0;
+  virtual double At (unsigned int output_index, unsigned int index,
+                     r2i::RuntimeError &error) = 0;
+
+  /**
+   * \brief Set predictions result
+   * \param data obtained from the model inference.
+   * \param size of the input data.
+   * \return Error code resulting from the set process.
```

Comment:
indicate data will be copied

Reply:
I think this will depend on the specific backend. For example, TF doesn't copy.

Reply:
I've reverted this change.